tests/testsuite/local_registry.rs

```diff
@@ -526,3 +526,75 @@ fn crates_io_registry_url_is_optional() {
     p.cargo("build").with_stderr("[FINISHED] [..]").run();
     p.cargo("test").run();
 }
+
+#[cargo_test]
+fn git_dependencies_do_not_require_a_checksum() {
```

Comment:
The title of the test is OK, but it may be better to give more contexts with some comments about why we skip checksum. Perhaps highlight the line calling `skip_checksum` which does some special stuff.

We could also provide a link to this pull request here as a reference for conveniences.